docs/5_endpoints.md

```diff
@@ -17,3 +17,20 @@ OAuth2 Proxy responds directly to the following endpoints. All other endpoints w
 - /oauth2/callback - the URL used at the end of the OAuth cycle. The oauth app will be configured with this as the callback url.
 - /oauth2/userinfo - the URL is used to return user's email from the session in JSON format.
 - /oauth2/auth - only returns a 202 Accepted response or a 401 Unauthorized response; for use with the [Nginx `auth_request` directive](#nginx-auth-request)
+
+### Sign out
+
+To sign the user out, redirect him to `/oauth2/sign_out`. However, this endpoint only removes oauth2_proxy's own cookies, i.e. the user is still logged in with the OIDC provider and will automatically re-login when accessing the application again. What you want to do is to redirect him to the OIDC provider's sign out page afterwards using the `rd` query parameter, i.e. redirect the user to something like (notice the url-encoding!):
+
+    /oauth2/sign_out?rd=https%3A%2F%2Fmy-oidc-provider.example.com%2Fsign_out_page
```

Comment:
This should be in a code block

Reply:
It is indented by 4 spaces and thus is a code block. Or did you mean "fenced code block"?